ci-operator/step-registry/single-node/conf/e2e/single-node-graceful-shutdown-conf-e2e-commands.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
```

Comment:
this will need to call the single-node-conf-e2e-commands.sh or copy that logic in... Not sure which is required here, but we need that section.

Reply:
oh this tweaks the e2e... works for me.